main/src/ca/uwaterloo/flix/runtime/Value.scala

```diff
@@ -342,6 +342,11 @@ object Value {
   def mkErr(v: AnyRef): Value.Tag = mkTag("Err", v)
 
   /**
+    * Returns the given Scala list `as` as a Flix set.
+    */
+  def mkFlixSet(as: List[AnyRef]): Value.Tag = mkTag("Set", mkList(as))
```

Comment:
For making Flix sets.  I used the name "mkFlixSet" because "mkSet" is already defined in this file (I'm not sure what it is used for there).